backend/dataall/core/environment/db/environment_repositories.py

```diff
@@ -303,6 +303,22 @@ def is_user_invited_to_environment(session, groups, uri):
         )
         return env_group is not None
 
+    @staticmethod
+    def is_group_invited_another_envs(session, organizationUri, group):
+        return (
+            session.query(EnvironmentGroup)
+            .join(Environment, EnvironmentGroup.environmentUri == Environment.environmentUri)
+            .filter(Environment.deleted.is_(None))
+            .filter(
+                and_(
+                    EnvironmentGroup.groupUri == group,
+                    Environment.organizationUri == organizationUri,
+                )
+            )
+            .count()
+            > 0
```

Comment:
nit: Don't need to specify >0 - if returns a value greather than 0 will evaluate as True in the if condition either way